javascript/javascript2/week3/Game/game.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Game</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="game.css">
+</head>
+
+<body>
+  <h1>Fastest presser</h1>
+  <input id="input" type="number" min="1" max="100">
+  <button id="startbtn" type="button">Start Game!</button>
+  <button id="restartbtn">ReStart Game!</button>
+  <h3 id="result"></h3>
+  <h3 id="alert"></h3>
+  <div class="row">
+    <div id="skey">
+      <h2>Press S</h2>
+      <h3 id="keys"></h3>
```

Comment:
Remember naming convention, I suggest to use `kebab case` fx: `key-s`

Reply:
👍🏻